plugins/GSdx/GSCrc.cpp

```diff
@@ -137,21 +139,23 @@ CRC::Game CRC::m_games[] =
 	// and as "Twisted Metal - Black" (PAL).
 	{0x788D8B4F, ICO, EU, 0},
 	{0x29C28734, ICO, CH, 0},
+	{0xC02C653E, GT4, CH, 0}, // China Speed!
```

Comment:
Sneaky little comment there xD

Reply:
You should make PR just to remove this line.